test/controllers/items_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class ItemsControllerTest < ActionDispatch::IntegrationTest
+  test "should get new" do
+    get items_new_url
+    assert_response :success
+  end
+
```

Comment:
should you check to make sure that you can access items that have already been added to your database?